lib/devise/secure_password/version.rb

```diff
@@ -1,5 +1,5 @@
 module Devise
   module SecurePassword
-    VERSION = '1.0.8'.freeze
+    VERSION = '2.0.0'.freeze
```

Comment:
@markeissler Why is this a bump by a major version when no breaking API changes have occurred, and the only thing that's changing is a test configuration?

Reply:
@petergoldstein We will be abandoning `Rails 5.0` with this as per policy (current supported and last supported versions). I just haven't removed the `spec/rails-app-5_0` directory yet.

It's a fair point, however. How about `1.1.0`?

Reply:
Sure.